pkg/version/version.go

```diff
@@ -27,6 +27,7 @@ func GetVersionInfo() map[string]string {
 	versionInfo["DDEV version"] = versionconstants.DdevVersion
 	versionInfo["cgo_enabled"] = strconv.FormatInt(versionconstants.CGOEnabled, 10)
 	versionInfo["global-ddev-dir"] = globalconfig.GetGlobalDdevDir()
+	versionInfo["go_version"] = runtime.Version()
```

Comment:
nit: for consistency with (most) of the other items, prefer go-version.

```suggestion
	versionInfo["go-version"] = runtime.Version()
```